.github/workflows/update-mirror.yml

```diff
@@ -0,0 +1,23 @@
+name: Update mirror
+
+on:
+  schedule:
+    - cron: '0 */2 * * *'
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout this repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Update mirror
+        run: |
+          BRANCH="$(git branch --show-current)"
```

Comment:
Shouldn't we exhibiting explicitly limit to master here?

Reply:
Scheduled workflows only run on the default branch, so `BRANCH` will always be `master`.  I mainly wrote the step this way because I was copying branch-agnostic code from elsewhere.

Reply:
thanks for the explanation!  If we go the "merge" way , then we need to pull (or fetch and merge) explicitly master into our default branch and push it and original master  (as described in https://github.com/datalad/datalad-extensions/issues/67 ).  

If we use `upstream/` namespace for original (upstream) git-annex branches here, then it might be a bit odd but would be, if we stay with `master` here:

```
git push --tags origin master upstream/master:upstream/master
```
I guess